gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -563,6 +563,8 @@
   private static final String TRANSLATED_SCHEMAS_DIR = "legacyPegasusSchemas";
   // Enable the use of argFiles for the tasks that support them
   private static final String ENABLE_ARG_FILE = "pegasusPlugin.enableArgFile";
+  // Flag used to skip running translate schema (pdl->pdsc) to save local build time
+  private static final String SKIP_TRANSLATE_SCHEMA = "skipTranslateSchema";
```

Comment:
1. The name of this property is misleading, one might think it applies when converting between PDL and PDSC. I suggest you rename it to something like `pegasusPlugin.disableLegacySchemaTranslation` or `pegasusPlugin.disableLegacySchemas`. I wanted to originally rename this task to this, but I couldn't for compatibility reasons.
2. I suggest you prefix this with `pegasusPlugin.` because I believe that's the best practice. If we run into an issue where one of our properties conflicts with a property from another plugin, that may be difficult to resolve.

Reply:
That's a good suggestion, I will update the name of this property.